pywbem/_subscription_manager.py

```diff
@@ -837,6 +841,18 @@ def add_filter(self, server_id, source_namespace, query,
             filter instance, or `None` if the `filter_id` parameter is
             specified.
 
+          source_namespace (:term:`string):
+            Optional source namespace of the indication filter. If the
+            parameter is provided the value will be inserted into the
+            CIM_IndicationFilter SourceNamespace property. Otherwise the
+            SourceNamespace property will not be created. The SourceNamespace
+            property is deprecated in the DMTF CIM_IndicationFilter class in
+            favor of SourceNamespaces( see the SourceNamespaces parameter
+            above). This parameter is provided only to support implementations
+            that use the keyword syntax with the SourceNamespace property
```

Comment:
I don't understand the first part of this sentence: "implementations that use the keyword syntax with the SourceNamespace property".

Is this about how the "source_namespace" parameter is passed to the Python function?

If so, then this supports "users" and not "implementations"; it is a parameter and not a property;  the parameter is named differently; and the whole story is that the change is backwards incompatible for users that specified it as a positional parameter and expected the SourceNamespace property to be set. I tend to think that whole part of the sentence should be removed, and a statement about the incompatible change should be made.

Reply:
Modified text